src/test/java/net/consensys/orion/enclave/sodium/SodiumEnclaveTest.java

```diff
@@ -13,22 +13,30 @@
 package net.consensys.orion.enclave.sodium;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import net.consensys.orion.enclave.EnclaveException;
 import net.consensys.orion.enclave.EncryptedKey;
 import net.consensys.orion.enclave.EncryptedPayload;
+import net.consensys.orion.enclave.PrivacyGroupPayload.Type;
 import net.consensys.orion.exception.OrionErrorCode;
 
 import java.security.Security;
+import java.util.Arrays;
 
+import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.crypto.sodium.Box;
+import org.apache.tuweni.crypto.sodium.Box.PublicKey;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class SodiumEnclaveTest {
+  private static final String PRIVACY_GROUP_ID1 = "negmDcN2P4ODpqn/6WkJ02zT/0w0bjhGpkZ8UP6vARk=";
```

Comment:
nit: I know the code exists in the enclave - but not sure I see these new tests are "enclave" related - not going to die on a hill about it however.

Reply:
moved into new test class